README.md

```diff
@@ -15,61 +21,307 @@ Our other repositories are:
+
+_NOTE: This project currently works with PostgreSQL version 15.x_
+
+```sh
+brew install postgresql@15
+```
+
+You'll now need to modify (or create, if it doesn't already exist) the `$PATH`
+environment variable to include the PostgreSQL binaries. Open the file you have
+worked with before to adjust your shell environment with the previous steps and
+do one of the following:
+
+If you already have a line that modifies the `$PATH` environment variable, just
+add this path into the existing string:
+
+```
+/opt/homebrew/opt/postgresql@15/bin
+```
+
+If you don't have a line for your `$PATH` environment variable, add it in like
+this, which will include the PostgreSQL binaries:
+
+```
+export PATH="/opt/homebrew/opt/postgresql@15/bin:$PATH
+```
+
+_NOTE: You don't want to overwrite your existing `$PATH` environment variable! Hence the reason why it is included on the end like this; paths are separated by a colon._
+
+#### Starting PostgreSQL and Redis
+
+With both PostgreSQL and Redis installed, you now need to start the services.
+Run this command so that they're available at all times going forward on your
+machine:
+
+```sh
+brew services start postgresql@15 redis
+```
+
+If they're already running, you can run this command instead to make sure the
+latest updates are applied to both services:
+
+```sh
+brew services restart postgresql@15 redis
+```
+
+### First-Time Project Setup
+
+Once all of pre-requisites for the project are installed and you have a
+cloud.gov account, you can now set up the API project and get things running
+locally!
+
+First, clone the respository in the directory of your choosing on your machine:
+
+```sh
+git clone git@github.com:GSA/notifications-api.git
+```
+
+Now go into the project directory (`notifications-api` by default) and set the
+local Python version, like this (assumes version Python `3.9.18` is what is
+installed on your machine):
+
+```sh
+cd notifications-api
+pyenv local 3.9.18
+```
+
+_If you're not sure which version of Python was installed with `pyenv`, you can check by running `pyenv versions` and it'll list everything available currently._
+
+Now [log into cloud.gov](https://cloud.gov/docs/getting-started/setup/#set-up-the-command-line)
+in the command line by using this command:
+
+```sh
+cf login -a api.fr.cloud.gov --sso
+```
+
+_REMINDER: Ensure you have access to the `notify-local-dev` and `notify-staging` spaces in cloud.gov_
+
+Now run the development Terraform setup by navigating to the development
+folder and running the script in it:
+
+```sh
+cd terraform/development
+./run.sh
+```
+
+In addition to some infrastructure setup, this will also create a local `.env`
+file for you in the project's root directory, which will include a handful of
+project-specific environment variables.
+
+**OPTIONAL:** If you want to send data to New Relic from your local develpment
+environment, set the `NEW_RELIC_LICENSE_KEY` environment variable within the
+`.env` file that was just created.
```

Comment:
Is anybody likely to need to do this? What would be the situation? If they aren't likely, we should just remove it from `sample.env` instead

Reply:
I'm definitely in favor of simplifying, so let's remove it; even if folks wanted to, they wouldn't necessarily be able to see as it is!